src/platform/Darwin/BUILD.gn

```diff
@@ -100,6 +100,7 @@ static_library("Darwin") {
   public_deps = [
     "${chip_root}/src/app:app_config",
     "${chip_root}/src/platform:platform_base",
+    "${chip_root}/third_party/inipp",
```

Comment:
Where is this dependency coming from?  I don't believe anything in src/platform/Darwin depends on inipp...

What's actually depending on it, and why is it not declaring the dependency?

Reply:
In particular, if the issue is that `src/controller/java` pulls in `ExamplePersistentStorage.cpp` and that uses inipp, then shouldn't the dependency be in `src/controller/java/BUILD.gn`?

Reply:
I was trying to mirror the scope that the Platform/Linux exposes the `inipp` library at. I can move both to `src/controller/java` if that is preferred.

Reply:
`platform/Linux` uses `inipp` itself in `src/platform/Linux/CHIPLinuxStorageIni.cpp`.  Please put the dependency in the thing that actually depends on `inipp`.

Reply:
Updated.